lib/views/watch_live/watch_live.dart

```diff
@@ -36,7 +29,7 @@ class _WatchLiveState extends State<WatchLive> {
               ),
               const SizedBox(height: 20),
               ElevatedButton( //change btn color
-                onPressed: navigateToLobby, 
+                onPressed: () => navigator.push(const LobbyScreen()), 
```

Comment:
Since the new function is a lot shorter, I think it makes sense to type it here directly instead of defining it elsewhere.